packages/web/config.js

```diff
@@ -33,6 +33,7 @@ if (typeof window === 'undefined' || process.env.APP_ENV === 'testing') {
 		ALGOLIA_SEARCH_KEY: process.env.ALGOLIA_SEARCH_KEY,
 		ALGOLIA_APPLICATION_ID: process.env.ALGOLIA_APPLICATION_ID,
 		ALGOLIA_INDEX: process.env.ALGOLIA_INDEX,
+		ALGOLIA_QUERY_SUGGESTIONS_INDEX: process.env.ALGOLIA_QUERY_SUGGESTIONS_INDEX,
```

Comment:
A quick question: shouldn't we include this for the other environments (`testing`, `staging`, `production`) as well?

Also, please, include it in the `.env.example` file 👍